facefusion/ffmpeg.py

```diff
@@ -218,9 +218,9 @@ def merge_video(target_path : str, output_path : str, temp_video_fps : Fps, outp
 	output_video_quality = state_manager.get_item('output_video_quality')
 	output_video_preset = state_manager.get_item('output_video_preset')
 	merge_frame_total = predict_video_frame_total(target_path, output_video_fps, trim_frame_start, trim_frame_end)
-	temp_video_path = get_temp_file_path(output_path)
+	temp_video_path = get_temp_file_path(state_manager.get_item('temp_path'), output_path)
 	temp_video_format = cast(VideoFormat, get_file_format(output_path))
-	temp_frames_pattern = get_temp_frames_pattern(output_path, '%08d')
+	temp_frames_pattern = get_temp_frames_pattern(state_manager.get_item('temp_path'), output_path, '%08d', state_manager.get_item('temp_frame_format'))
```

Comment:
'%08d' should be in the end, try to follow the same order of arguments over similar or related methods